src/components/clientDashboard/enrollments/ClientAssessments.tsx

```diff
@@ -54,7 +62,8 @@ const ClientAssessments = () => {
           GetClientAssessmentsQueryVariables,
           ClientAssessmentType
         >
-          showFilters
+          showTopToolbar
+          filters={filters}
```

Comment:
This feels a little verbose. Shouldn't the "top tool bar" show up automatically if `filters` are provided? Or are there cases where we want to pass filters but for some reason want to hide them...? I don't think so.